README.md

```diff
@@ -259,6 +258,74 @@ curl http://aws-java-sample-java-function.default.dev.triggermesh.io -d '{"event
 {"message":"Hello, the current time is Tue Apr 07 13:59:17 GMT 2020"}
 ```
 
+### Run in Docker 
+
+For cases in which the use of additional components (tm CLI, Tekton, Knative, k8s) is undesirable, it is possible to build KLR function as a standalone Docker container and run it in any environment. To do this, you should extract Dockerfile from the runtime you are interested in, put it in the directory with your function, update the handler variable, and build the container. Here is Dockerfile definition for all runtimes:
+
+- [go](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/go-1.x/runtime.yaml#L44-L68)
```

Comment:
these links will break when we make additional commits right ?

Reply:
not really, that's why I pointed to the particular commit instead of a head - these links will stay relevant despite any changes in the master. We may need to update these links in case of important, incompatible changes, but for now, it should work just fine.